packages/tools/cookbook/package.json

```diff
@@ -10,7 +10,10 @@
     "_phase:build": "",
     "_phase:test": "",
     "build": "",
+    "dev:postinstall": "rushx pactjs:retrieve:contract; rushx pactjs:generate:contract",
```

Comment:
This worked after running `mkdir ./contracts` for me. Maybe it can be 
```
"dev:postinstall": "mkdir -p ./contracts; rushx pactjs:retrieve:contract; rushx pactjs:generate:contract",
```